### DISCUSSION TITLE
Can I juest used wizard in O3D312 to get the the  inner diameter, outer diameter,  height and volume of difference tires？

### DISCUSSION THREAD

**Discussion by mikufans-dumo** [ifm/ifm3d#463] [Q&A] [answered]
I need  the data (inner diameter, outer diameter,  height and volume of difference tires) to guide  the robotic arm to grab the tire to the correct position.
Can I juest used wizard in O3D312 to get the the  inner diameter, outer diameter,  height and volume of difference tires？

**Comment by desengph** [comment]
Hi @mikufans-dumo,
thank you for your message. We will follow up on your questions asap.

**Reply by debertst** [reply, marked as answer]
Hi @mikufans-dumo, 
This should be possible with the Robot Pick and Place Wizard, provided that the tires are separated or have a minimum distance of 4 cm between them.  This will give you the X,Y,Z coordinates, the rotation and the height, length and width of the object. To get the wall thickness of a tire you have to switch to the user defined mode in the wizard and add the wall thickness in the interface settings. 

![image](https://github.com/user-attachments/assets/b93f6fce-e7a0-4dc7-ba61-fdbf2446a3c7)

**Reply by mikufans-dumo** [reply]
OK,thanks for your reply.I will try it later.
